charts/alfresco-activiti/Chart.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v2
 description: A Helm chart for Alfresco Activiti
 name: alfresco-activiti
-version: 0.1.1
+version: 0.2.0-alpha.0
```

Comment:
Why not a major bump here like in the other charts?

Reply:
This is because in the previous version the `affinity` value was already a dict so we're not breaking the configuration "interface", hence a minor bump is enough